datanode/gateway/graphql/schema.graphql

```diff
@@ -1709,6 +1709,8 @@ type Market {
     dateRange: DateRange,
     "Pagination information"
     pagination: Pagination
+    "Filter orders"
+    filter: OrderFilter
```

Comment:
Ditto above; I would have thought this is generally for the case where you've got an order ID and you want to see it's history, I can't really think of a use-case for wanting to filter it further; though I guess it doesn't do any harm :)

Reply:
I'm sorry, I'm a bit confused, the ordersConnection under markets and parties lists the current orders for the market or party. It doesn't let you filter for a specific order and get the order's history. The filtering here should be valid.

There is a gRPC function that allows you to retrieve the order history for a specific order ID, which is exposed in GraphQL the orderVersionsConnection that doesn't have any filtering but lets you look up the history of an order using it's orderID.

Reply:
I think I understand where the confusion is. The way GH is displaying this makes it look like the filter is being applied to orderVersionsConnection, but it's not. The filter is only applied to ordersConnection under the partiesConnection and marketsConnection query.